tests/integration/test_logcollector/test_macos/test_macos_log_process.py

```diff
@@ -142,7 +143,7 @@ def test_macos_log_process_stop_suddenly_warning(get_configuration, configure_en
 
     macos_logcollector_monitored = logcollector.callback_monitoring_macos_logs
     log_monitor.start(timeout=logcollector.LOG_COLLECTOR_GLOBAL_TIMEOUT, callback=macos_logcollector_monitored,
-                            error_message=logcollector.GENERIC_CALLBACK_ERROR_TARGET_SOCKET)
+                      error_message=logcollector.GENERIC_CALLBACK_ERROR_TARGET_SOCKET)
     time.sleep(5)
```

Comment:
Hardcoded sleeps are not allowed, we should create a variable with a descriptive name.

Reply:
Done 9c61d46